tests/layer_tests/pytorch_tests/pytorch_layer_test_class.py

```diff
@@ -146,7 +146,7 @@ def use_torch_compile_backend():
                 if np.array(cur_fw_res).size == 0:
                     continue
                 cur_ov_res = infer_res[compiled.output(i)]
-                print(f"fw_res: {cur_fw_res};\n ov_res: {cur_ov_res}")
+                assert cur_ov_res.shape == cur_fw_res.shape, "Shapes are not equal: {} vs {}".format(cur_ov_res.shape, cur_fw_res.shape)
```

Comment:
can we slightly extend error message?

I always forget in which place located pytorch tensor and where ov :) It is also will be useful to add index of output to simplify debugging